vllm/logger.py

```diff
@@ -26,15 +36,23 @@ def format(self, record):
 _default_handler = None
 
 
+def _get_default_logging_level():
+    verbosity = os.getenv("VLLM_VERBOSITY", "info").lower()
```

Comment:
For vLLM logs, we allow `debug`, `info`, `warning`, `error` and `critical` levels.
The default is `info`. The default can be overridden by `VLLM_VERBOSITY` variable.

Users may want to set it to `warning` in production.